### PR TITLE
10.0 mig pos quick logout

### PR DESCRIPTION
**Pull request by cvinh**

### CODE REVIEW

File: pos_quick_logout/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 {
     'name': 'Point of Sale - Quick Logout',
-    'version': '9.0.1.0.0',
+    'version': '1.0.0',
```
Comment (legalsylvain) [inline pos_quick_logout/__manifest__.py]: please set '10.0.1.0.0'

File: pos_quick_logout/models/pos_quick_logout.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```
Comment (legalsylvain) [inline pos_quick_logout/models/pos_quick_logout.py]: please rename the file into pos_config.py

File: pos_quick_logout/__manifest__.py (hunk 1 above)
Comment (cvinh) [reply pos_quick_logout/__manifest__.py]: done in https://github.com/OCA/pos/pull/197/commits/262f4153487c2e792c8b07f066a2a2bcafe44d19

File: pos_quick_logout/models/pos_quick_logout.py (hunk 2 above)
Comment (cvinh) [reply pos_quick_logout/models/pos_quick_logout.py]: done

**Comment by legalsylvain** [comment]
hi thanks ! 
sorry, but the same for the name of the view. the convention is /views/model_name_view.xml

thanks.

**Comment by cvinh** [comment]
name for the view is /views/pos_quick_logout_view.xml indeed

**Comment by legalsylvain** [comment]
yes, but it should be renamed.
git mv pos_quick_logout/views/pos_quick_logout_view.xml pos_quick_logout/views/pos_config_view.xml

regards.

ref : https://github.com/OCA/maintainer-tools/tree/master/template/module/views

**Comment by cvinh** [comment]
of course my bad, I read modUlE_name_view.xml instead of modEl_name_view.xml

**Comment by cvinh** [comment]
@legalsylvain could you help with travis problem ?

**Comment by legalsylvain** [comment]
you can click on detail, on travis. 
here this is flake8 issue.
https://travis-ci.org/OCA/pos/jobs/271762288#L728

you have two useless import

**Comment by cvinh** [comment]
@legalsylvain thanks !!

**Comment by legalsylvain** [comment]
could you squach your commits from ddfc82f to 5e5dd58 ? 
(it will make review more easy).

regards.

**Comment by cvinh** [comment]
@legalsylvain rebase is done

**Review by legalsylvain — APPROVED** [review]
Test and code review. works like a charm. Thanks !

**Pull request merged**